src/pages/App.js

```diff
@@ -63,7 +64,32 @@ export default function App() {
       />
       <button type="submit" onClick={handleSubmit}>Search</button>
       <p>
-        {foods.map((elem) => <li>{elem.description}</li>)}
+        {foods.slice(10, 30).map((elem) => {
+          if (elem.description.toLowerCase() === food.toLowerCase()) {
+            return null;// no need to show it in the page
+          }
+          if (!set.has(elem.description.toLowerCase())) {
+            set.add(elem.description.toLowerCase());
+            return (
+              <>
+                <li>{elem.description}</li>
+                <p> These are the nutrations that are present: </p>
+                <div>
+                  {elem.foodNutrients.map((element) => (
+                    <p>
+                      {element.nutrientName}
+                      =
+                      {element.value}
+                      g
```

Comment:
Is the unit guaranteed to be grams? I think I remember seeing a "unit" string included in the response, but maybe that was for something else.

Reply:
Yea you right I will fix it right now